### ISSUE TITLE
Benchmarking with the volume mount performance

### ISSUE THREAD

**Issue by yxliang01** [chriskuehl/docker-storage-benchmark#5]
@chriskuehl Thanks for the interesting benchmarking. I think it should be very interesting to see also for the performance of docker containers on the mounted volume. :) Are you considering do so?

**Comment by asottile** [comment]
I believe you're asking about the first benchmark included here: https://github.com/chriskuehl/docker-storage-benchmark#reading-files  "read-file-tree-mounted"?

**Comment by yxliang01** [comment]
@asottile Yeah, but it's only one (read file tree), I am thinking it would be good if there's full set of benchmarking for the mount case.
